src/arrayMethodJoin.js

```diff
@@ -5,7 +5,25 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let result = '';
+    let newSeperator = separator;
```

Comment:
```suggestion
    const newSeparator = separator;
```
after  fix prev comment about default argument change let on const and fis typo